pyproject.toml

```diff
@@ -1,5 +1,166 @@
-# pyproject.toml is currently only used to configure developement tools.
-# Configurations for the lightly package are in setup.py.
+[build-system]
+requires = [
+    "setuptools>=21",
+    "setuptools-scm"
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name="lightly"
+requires-python = ">=3.6"
+authors = [
+    {name = "Lightly Team", email = "team@lightly.ai"},
+]
+license = {file = "LICENSE.txt"}
+description="A deep learning package for self-supervised learning"
```

Comment:
The `long_description` got lost. Does `pyproject.toml` support it?

Reply:
I believe the following is all that we can do in a `pyproject.toml`:

```toml
dynamic = ["version", "readme"]
# ...
[tool.setuptools.dynamic]
readme = {file = ["README.md"], content-type = "text/markdown"}
```